db/seeds.rb

```diff
@@ -6,24 +6,24 @@
 chow = Category.create(name: 'Chow', image: "fa fa-cutlery")
 shop_all = Category.create(name: 'Shop All', image: "fa fa-archive")
 
-item1 = brewing.items.create!(title: 'Mexican Organic Dark Coffee', description: 'Bold smoky aroma with a smooth body and dark cocoa finish', price: 1200, photo: 'beans.png')
-item2 = brewing.items.create!(title: 'Columbian Dark Coffee', description: 'Creamy and smooth-bodied with a smoky aroma and well-balanced flavor', price: 1500, photo: 'coffee.bag.jpg')
+item1 = brewing.items.create!(title: 'Mexican Organic Dark Coffee', description: 'Bold smoky aroma with a smooth body and dark cocoa finish', price: 1200, image: File.new("#{Rails.root}/app/assets/images/beans.png"))
+item2 = brewing.items.create!(title: 'Columbian Dark Coffee', description: 'Creamy and smooth-bodied with a smoky aroma and well-balanced flavor', price: 1500, image: File.new("#{Rails.root}/app/assets/images/coffee.bag.jpg"))
```

Comment:
Useless assignment to variable - `item2`.<br>Line is too long. [226/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.